src/main/java/ca/corefacility/bioinformatics/irida/ria/web/services/UIPipelineService.java

```diff
@@ -74,8 +92,21 @@ public UIPipelineDetailsResponse getPipelineDetails(UUID id, Locale locale) thro
          */
 		detailsResponse.setSavedPipelineParameters(getSavedPipelineParameters(workflow, locale));
 
-		return detailsResponse;
-	}
+		/*
```

Comment:
Fix formatting of this comment

Reply:
What's wrong with it?

Reply:
It was a multi-line comment. Sorry about that